doc/rest.md

```diff
@@ -130,6 +130,7 @@ Body parameters:
       - JPEG (.jpg, .jpeg, with .jpg.aux.xml, .jgw, .jpgw, .jpegw or .wld)
       - any of above types in single ZIP file (.zip)
       - file names, i.e. array of strings
+      - more than one file is allowed for raster files in combination with *time_regex* parameter
```

Comment:
more than one main raster file is allowed in combination with *time_regex* parameter